src/app.rs

```diff
@@ -2281,8 +2260,8 @@ pub(crate) enum Dispatch {
         scope: Scope,
         if_current_not_found: IfCurrentNotFound,
     },
-    OpenFile(CanonicalizedPath),
-    OpenFileFromPathBuf(PathBuf),
+    OpenFile(CanonicalizedPath, BufferOwner, bool),
+    OpenFileFromPathBuf(PathBuf, BufferOwner, bool),
```

Comment:
Try to avoid tuples if there are more than 1 parameters, use key-value instead.

Luckily this can be done easily via Code Actions (`space c`) when using Rust Analyzer LSP.